report-jtreg/src/main/resources/io/jenkins/plugins/report/jtreg/JenkinsReportJckGlobalConfig/config.jelly

```diff
@@ -94,9 +94,9 @@
                         Track a failed test with the list tool
                         <ul>
                             <li><i>Link label:</i> track</li>
-                            <li><i>Base tool page:</i> list.html?=</li>
-                            <li><i>Spliterator:</i> -</li>
-                            <li><i>Tool arguments:</i> <code>-view=all-tests, -output=html, -fill, -track=%{TESTNAME}  %{JOBNAME}  0, -365</code></li>
+                            <li><i>Base tool page:</i> list.html?generated-part=&amp;custom-part=</li>
+                            <li><i>Spliterator:</i> [.-]</li>
```

Comment:
This do not seem right. "." is everything. So maybe "\\." ?

Reply:
Afaik, when the symbols are inside of brackets, it takes the symbol as is, so the dot is really a dot, not everything. It at least works with the comparator links

Reply:
hmm.. So what do you think `[\.-]` did? I guess `[\\.-]` isimply changed to three chars of  \\ . and -.  But  with  `[\.-]` I'm not sure.

Reply:
> Afaik, when the symbols are inside of brackets, it takes the symbol as is, so the dot is really a dot

Gosh. I really did not know that. I did few checks and you are right. Where did you found that written?

Reply:
yah... I have not found it in spec... But it seems it is indeed how it works.. thanx again!